skimage/restoration/_denoise.py

```diff
@@ -341,17 +274,11 @@ def denoise_tv_chambolle(im, weight=0.2, eps=2.e-4, n_iter_max=200,
     if not im_type.kind == 'f':
         im = img_as_float(im)
 
-    if im.ndim == 2:
-        out = _denoise_tv_chambolle_2d(im, weight, eps, n_iter_max)
-    elif im.ndim == 3:
-        if multichannel:
-            out = np.zeros_like(im)
-            for c in range(im.shape[2]):
-                out[..., c] = _denoise_tv_chambolle_2d(im[..., c], weight, eps,
-                                                       n_iter_max)
-        else:
-            out = _denoise_tv_chambolle_3d(im, weight, eps, n_iter_max)
+    if multichannel:
+        out = np.zeros_like(im)
+        for c in range(im.shape[2]):
```

Comment:
Can we also test for roughly consistent levels of denoising as dimensions
increase?